### PR TITLE
[Merged by Bors] - Updated to operator-rs 0.10.0, added scraping label

### PR DESCRIPTION
**Pull request by maltesander**
## Description

fixes https://github.com/stackabletech/trino-operator/issues/103

- updated to operator-rs 0.10.0
- added scraping label

TODO:
- [x] Switch to regorule tag

## Review Checklist
- [ ] Code contains useful comments
- [ ] (Integration-)Test cases added (or not applicable)
- [ ] Documentation added (or not applicable)
- [ ] Changelog updated (or not applicable)
- [ ] Cargo.toml only contains references to git tags (not specific commits or branches)
- [ ] Helm chart can be installed and deployed operator works (or not applicable)

Once the review is done, comment `bors r+` (or `bors merge`) to merge. [Further information](https://bors.tech/documentation/getting-started/#reviewing-pull-requests)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+### Added
+
+- monitoring scraping label `prometheus.io/scrape: true` ([#118]).
```
Comment (teozkr) [inline CHANGELOG.md:9]:
```suggestion
- Monitoring scraping label `prometheus.io/scrape: true` ([#118]).
```

File: rust/crd/Cargo.toml [hunk 2]
```diff
@@ -8,8 +8,8 @@ repository = "https://github.com/stackabletech/trino-operator"
 version = "0.3.0-nightly"
 
 [dependencies]
-stackable-operator = { git = "https://github.com/stackabletech/operator-rs.git", tag = "0.9.0" }
-stackable-regorule-crd = { git = "https://github.com/stackabletech/regorule-operator.git", tag = "0.5.0" }
+stackable-operator = { git = "https://github.com/stackabletech/operator-rs.git", tag = "0.10.0" }
+stackable-regorule-crd = { git = "https://github.com/stackabletech/regorule-operator.git", branch = "main" }
```
Comment (teozkr) [inline rust/crd/Cargo.toml:12]: Probably want to do a release here?

Comment (maltesander) [reply rust/crd/Cargo.toml:12]: yeah its in the TODO. We have problems with the release script. Will stay draft until regorule was released.

**Review by razvan — APPROVED** [review]
ok

**Pull request closed without merging**